test/runner_isolation.sh

```diff
@@ -13,4 +13,4 @@ shift
 
 $ISOLATIONTEST "$@" | \
    sed -e 's!_[0-9]\{1,\}_[0-9]\{1,\}_chunk!_X_X_chunk!g'
- 
+
```

Comment:
Spurious change, might be good to not add to this commit since this makes finding problems later more difficult.

Reply:
Yes, I missed this, will remove it completely.